sqltest/sql_test_runner.go

```diff
@@ -194,6 +196,13 @@ func (w *sqlTestsuite) setupPranaCluster() {
 		}
 		cnf.SourceStatsEnabled = true
 		cnf.ProtobufDescriptorDir = ProtoDescriptorDir
+
+		cnf.IntraClusterTLSConfig.Enabled = true
```

Comment:
We don't want to turn on intra cluster TLS for *all* SQL tests, it should be controlled by a parameter in runSQLTest(). I.e. add a new param useIntraClusterTLS like we do for useHTTPAPI

Reply:
Done